core/templates/components/profile-link-directives/profile-link-image-backend-api.service.ts

```diff
@@ -36,6 +36,8 @@ export class ProfileLinkImageBackendApiService {
 
   fetchProfilePictureData(profileImageUrl: string): Promise<string> {
     return this.http.get<ProfileDict>(profileImageUrl).pipe(
+      // A URL encoded base64 image was treated as unsafe by Angular.
```

Comment:
Explan this more, and in general terms. E.g.:

We decode the image because the values passed from the backend contain "%" characters (due to escaping), which are treated as unsafe by Angular when they are part of URLs.